cpp/Platform.Interfaces/CList.h

```diff
@@ -13,18 +13,18 @@ namespace Platform::Interfaces
 {
     namespace Internal
     {
-        template<typename TRawSelf, typename... TItem>
+        template<typename TRawSelf, typename... TItems>
```

Comment:
Why `TItems`? It's an optional type

Reply:
It is alone

Reply:
But https://github.com/linksplatform/Interfaces/pull/85#discussion_r739678867